uberenv.py

```diff
@@ -262,6 +255,17 @@ def parse_args():
                       default=False,
                       help="Only install (using pre-setup Spack).")
 
+    # spack environment directory
+    parser.add_option("--spack-env",
```

Comment:
This option is going to confuse people. We need to think of a better name.  Thoughts, @kennyweiss ?

Reply:
Maybe `spack-env-name` ?

Reply:
LiDO, for another reference, calls the spack environment directory `venv_dir` and the yaml file defining the spack environment `manifest_file`. Maybe something like that would be good too.

`spack_env -> venv_dir : Spack environment directory`
`spack_env_file -> manifest_file : YAML file defining spack environment`

Reply:
How is this used along with `--prefix`?  We need an extra name?

Reply:
If we do need to control the name, maybe: `--spack-env-dir-name`

Reply:
`--prefix` is where uberenv does all it's work (downloads spack, creates logs, also where axom/serac's install prefix lives at this level though this is controlled by the spack config.yaml).

`--spack-env-name` (or whatever) is the name of the environment but Spack also creates a directory that holds a lot of information (logs, a spack.yaml, a package repository, bunch of metadata, etc). More info here:

https://spack.readthedocs.io/en/latest/environments.html#creating-a-named-environment

Reply:
I would say `--spack-env-name` sounds good.

Does it have a default (in case you don't care)?

Reply:
It defaults to `spack_env`

Reply:
I'll change it to `--spack-env-name`, then